insonmnia/locator/config.go

```diff
@@ -3,32 +3,48 @@ package locator
 import (
 	"time"
 
+	"fmt"
 	"github.com/jinzhu/configor"
 	"github.com/sonm-io/core/accounts"
 )
 
-type LocatorConfig struct {
+type StoreConfig struct {
+	Type     string `required:"true" default:"boltdb" yaml:"type"`
+	Endpoint string `required:"true" default:"/tmp/sonm/boltdb" yaml:"endpoint"`
+	Bucket   string `required:"true" default:"sonm" yaml:"bucket"`
+	//	KeyFile  string `yaml:"key_file"`
```

Comment:
Looks like you copy paste code, commenting some strings. Can we reuse it?

Reply:
It was decided to add implementation for bolt db only which has no support for Certs/Keys, so these settings are not used  it may be deleted safely, however they are necessary if consul/etcd implementation is added later.